### PR TITLE
Created user authentication Java servlet and modified various XML files

### PR DESCRIPTION
**Pull request by LilCSharp**
### What is a quick description of the change?

I developed a Java servlet that guides the user to a login page, and then redirects them back to a page following login and logourt
The login screen guides the user to a Google login page where they can enter their account. There is also a home page link that
allows the user to leave the login page as well.

### Screenshot/Screengrab after this change?

https://user-images.githubusercontent.com/50273061/85454439-f03e5880-b56a-11ea-9f4d-47ced7f22702.png

### Is this fixing an issue?

The XML modifications are supposed to allow the development testing server to be run, but they have caused only errors on my part, so I had to use my personal project for testing. The Java servlet completely constructs the authentication system, assuming similar behavior.

### Are there more details that are relevant?

There are multiple links in the Java servlet that will have to be filled in as the UI is constructed. These links include: the home page
that the "Home" button refers to, and the page that the app will redirect to once login or logout is successful.

### CODE REVIEW

**Review by jamierashley — COMMENTED** [review]
Maybe make the UserAuthServlet and LoginServlet the same name

**Review by jamierashley — APPROVED** [review]

**Review by aliciasimon — APPROVED** [review]
LGTM

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,55 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
```
Comment (antrromet) [inline pom.xml]: Could you do a pull?

This file was already added by Jamier.

File: target/step13-2020-1/WEB-INF/appengine-web.xml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (antrromet) [inline target/step13-2020-1/WEB-INF/appengine-web.xml:1]: Why is the target folder name "step13-2020-1"?

and not just "step13-2020"

Comment (LilCSharp) [reply target/step13-2020-1/WEB-INF/appengine-web.xml:1]: It must be named that for the pom.xml file to work. The folder format stands for the project name and the version number. When Maven is run, it needs to search for the Web XML files in a folder of that format.

**Review by antrromet — APPROVED** [review]
As I mentioned in the previous review, you'll have to do a sync before merging.

**Pull request merged**